src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/Example3.java

```diff
@@ -16,10 +16,12 @@ class Example3 {
 
   public void method0() {
     int mode = 0;
+    int x;
     switch (mode) {
       case 1:
-        int x = 1;
+        int y = 1;
         break;
+      case 2: {x = 1;}   // violation '}' at column 22 should be alone on a line'
```

Comment:
Add another example with code before the `{}` to show no violation.

Reply:
I am leaning to having some verbiage explain how {}s in case works.

Reply:
@rnveach done and added  explanation in a comment beside the example

Reply:
@rnveach ping